consensus/consensus.go

```diff
@@ -24,8 +25,12 @@ const (
 
 // Consensus is the main struct with all states and data related to consensus process.
 type Consensus struct {
+	// Decider handles whether quorum achieved on the shard this consensus running on and decides on
+	// distribution of blockrewards - Both parameterized by quorum.Policy
 	Decider quorum.Decider
-
+	// CommitteeAssigner provides information about who the SuperCommittee members are, their public
+	// keys
+	CommitteeAssigner committee.MemberReader
```

Comment:
Why not CommitteeReader?

Reply:
Better, assigner was older name I thought of using; done.